app/scripts/modules/core/src/task/task.read.service.ts

```diff
@@ -26,17 +26,13 @@ export class TaskReader {
     return API.one('tasks', taskId)
       .get()
       .then((task: ITask) => {
-        OrchestratedItemTransformer.defineProperties(task);
```

Comment:
this seemed to be entirely unnecessary, but it's theoretically possible that doing this transform, then the potential `task.execution` transform, then a second identical transform is somehow necessary. I dunno.